pkgs/top-level/all-packages.nix

```diff
@@ -23984,9 +23984,7 @@ with pkgs;
   ngtcp2 = callPackage ../development/libraries/ngtcp2 { };
   ngtcp2-gnutls = callPackage ../development/libraries/ngtcp2/gnutls.nix { };
 
-  nix-plugins = callPackage ../development/libraries/nix-plugins {
-    nix = nixVersions.nix_2_16;
```

Comment:
We should keep the pin but bump it to 2_17, as otherwise it would still break some other day.

Reply:
Good point